src/nat/runtime/session.py

```
@@ -161,6 +162,31 @@ def set_metadata_from_http_request(self, request: Request) -> None:
+                        trace_id_int = uuid.UUID(trace_id_hex).int
+                        self._context_state.workflow_trace_id.set(trace_id_int)
+            except Exception:
+                pass
+
+        if not self._context_state.workflow_trace_id.get():
+            workflow_trace_id = request.headers.get("workflow-trace-id")
+            if workflow_trace_id:
+                try:
+                    self._context_state.workflow_trace_id.set(uuid.UUID(workflow_trace_id).int)
+                except Exception:
+                    pass
```

Ruff (0.13.2)

`try`-`except`-`pass` detected, consider logging the exception

(S110)